website/sidebars.json

```diff
@@ -119,9 +119,9 @@
       "operator-docs/discovery-filtering",
       "operator-docs/bulk-operations",
       "operator-docs/cmr-operations",
-      "operator-docs/discovery-filtering",
-      "operator-docs/rerunning-workflow-executions",
-      "operator-docs/replay-kinesis-messages"
+      "operator-docs/naming-executions",
+      "troubleshooting/rerunning-workflow-executions",
+      "features/replay-kinesis-messages"
```

Comment:
Again, not really related to my changes but noticed issues with these pages showing up in the docs when testing locally